.circleci/Dockerfile

```diff
@@ -53,9 +53,16 @@ RUN  sudo apt-get -qqy install ruby ruby-dev rubygems build-essential && \
      sudo gem install --no-ri --no-rdoc fpm
 
 # Include minikube and kubectl in the image
-RUN curl -Lo /tmp/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.7.4/bin/linux/amd64/kubectl && \
```

Comment:
thanks for parametrizing this and others hardcoded version in the original file!